content/docs/user-guide/gems/reference/aws/aws-gamelift/resource-management.md

```diff
@@ -165,18 +203,34 @@ $ cdk synth
 
 ### Enable optional features
 
+
+#### Upload with support stack
+
 If you need the AWS CDK application to upload a local package and create a GameLift build accordingly, you must enable the `upload-with-support-stack` context variable:
 
 ```cmd
 $ cdk synth -c upload-with-support-stack=true --all
 ```
 
+
 When this optional feature is enabled, an additional CloudFormation stack will be deployed. The additional stack contains the AWS resources that are required to support the build upload and creation. The `--all` argument tells the CDK application to synthesize all the available stacks.
 
-Optionally, you can create a game session queue resource using this CDK application by providing the `create_game_session_queue` context variable when synthesizing stack(s). The following example command synthesizes the application with all the optional features enabled:
+
+#### Create game session queue
+
+Optionally, you can create a game session queue resource using this CDK application by providing the `create_game_session_queue` context variable when synthesizing stack(s). The following example command synthesizes the application with this optional features enabled:
```

Comment:
Its is recommended you create the optional game session queue.... We should also recommend async session creation behind a queue. See https://docs.aws.amazon.com/gamelift/latest/developerguide/queues-best-practices.html